README.rst

```diff
@@ -141,6 +141,28 @@ Project resources
 - `Download development snapshot <https://github.com/mopidy/mopidy-spotify/tarball/develop#egg=Mopidy-Spotify-dev>`_
 
 
+Helping out
+===========
+
+The following commands should get you going.::
+
+    git clone https://github.com/mopidy/mopidy-spotify.git
+    cd mopidy-spotify
+
+    # start editing files
+
+    # install your changes
+    sudo python setup.py install
```

Comment:
Thanks for the insights, will update.
Not very familiar with python over here...